### ISSUE TITLE
Unable to see the JSPM under dependencies in Visual Studio 2015

### ISSUE THREAD

**Issue by varghesep** [jspm/jspm-cli#1002]
I'm currently using Aurelia which uses JSPM. I'm able to see the packages managed by **Bower** and **NPM**, but not JSPM.

How can I see JSPM packages through the VS IDE? 

![jspm](https://cloud.githubusercontent.com/assets/12233474/9154522/f2b36238-3e58-11e5-984d-861cef7f55c9.png)

**Comment by guybedford** [comment]
Did you manage to get this to work ok?

**Comment by varghesep** [comment]
I was not able to get the JSPM under the Dependencies section. I think the ASP.NET 5 project .prj needs to be modified to achieve this.

I also struggle with the performance of VS 2015 now because some processes going on in the background which puts VS 2015 into NOT RESPONDING for minutes. This happens constantly. I turned off ReSharper which did not help. I don't know is this caused by JavaScript Language Service which runs in the background to alert the syntax errors. I started seeing this as more jspm packages are added under **wwwroot**.
